### PR TITLE
Initial Pull Request

### PR DESCRIPTION
**Pull request by alexjurgensen**
This is the first pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Digita11y",
+  "version": "1.0.0",
+  "description": "An app for accessing Roundware servers on iOS.",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accessibilityhound/roundware-ios-access.git"
```
Comment (eosrei) [inline package.json]: This package information is incorrect.

Comment (alexjurgensen) [inline package.json]: Any suggestions on what changes I should make to make it more accurate?

Comment (eosrei) [inline package.json]: The repo is `https://github.com/roundware/roundware-ios-access.git`

Plus, the name *Digita11y*, is this the name of your app? If so, it should be stored in a separate repository. This app is supposed to be the generic app version. I might be wrong please confirm with @hburgund.

**Comment by eosrei** [comment]
Overall, I'm not sure I understand why this is being submitted as a pull request at this time. There's only 37 lines (in 4 commits?) without any working code and the pull request itself has no description. Is this basic structure required for other people to get started?

BTW Hi! I'm Brad. I've done months of work on the RW Server and am currently working on it and the RW Android codebase. I'm interested to see where the iOS app is headed.

**Comment by sinabahram** [comment]
Brad, @hburgund can explain further, but this is the app for RoundWare, yes. We might need to spin up a repository called Digita11y, or rename this one. Considering there's no code. Do whatever makes more sense. I don't believe I have access rights to do so.

We are doing a series of pull requests not targeted at any features or specific issues for the first week to two weeks just to get the basics of the app in there, and also to make sure all parts of the process work. We will then be packaging pull requests to match against specific github issues, bug fixes, etc.

Given the rather compressed nature of the timeline, we're hoping not to block on these issues too long, so hopefully this can be resolved within the next 12 hours or so.

Thanks in advance for understanding.

**Comment by hburgund** [comment]
The project as supported by IMLS has been named *Digita11y* and is itself intended to be a generic app that museums, cultural orgs etc will be able to modify/customize to their liking and in doing so, re-brand.  

So though *Digita11y* sounds like the name of a specific app, it is actually the name of the generic starting point app.  That said, as we develop this app, we may find the need to include some test/sample data/assets in order to properly demonstrate the various functionalities, but it will be as generic as is reasonable.

This name did not exist when the repo was initially created, but now that it does, I have changed the repo name to reflect it properly.

**Comment by eosrei** [comment]
@sinabahram Halsey can add @alexjurgensen as a repo committer. If you are trying to build this quickly you could save time by avoiding pull requests and simply push to the repo directly for the timebeing. Pull requests seems like an added layer of hassle at this point.

**Comment by sinabahram** [comment]
That’s definitely true. We’re going to transition to pull requests per features soon, so we figured this would at least make sure that part of the flow works.

**Pull request merged**